cupy/statistics/histogram.py

```diff
@@ -79,6 +81,10 @@ def histogram(x, bins=10):
 
     y = cupy.zeros(bins.size - 1, dtype='l')
     _histogram_kernel(x, bins, bins.size, y)
+
+    if density:
+        db = cupy.array(cupy.diff(bins), float)
```

Comment:
I think `float` is not specific enough.
dtype='float64' better?

Reply:
Thanks. It is a separate question whether the first argument's name should be changed from `x` to `a` to match NumPy.